includes/class-wcsatt-core-compatibility.php

```diff
@@ -48,7 +55,7 @@ private static function get_wc_version() {
 	 * @return boolean
 	 */
 	public static function is_wc_version_gte_2_7() {
```

Comment:
These `_MAJOR_MINOR` functions are probably unnecessary and could be deprecated/removed. Are we dropping support for WC 2.6, after all? If so, then we don't need them anyway.